subprojects/settings.gradle.kts

```diff
@@ -99,31 +92,36 @@ include(":common:truth-extensions")
 include(":common:composite-exception")
 include(":common:throwable-utils")
 include(":common:coroutines-extension")
-include(":common:test-report-api")
-include(":common:test-report-dsl-api")
-include(":common:test-report-dsl")
 
 include(":android-test:resource-manager-exceptions")
 include(":android-test:websocket-reporter")
 include(":android-test:keep-for-testing")
 include(":android-test:ui-testing-maps")
 include(":android-test:ui-testing-core-app")
 include(":android-test:ui-testing-core")
-include(":android-test:test-report")
-include(":android-test:test-inhouse-runner")
 include(":android-test:instrumentation")
-include(":android-test:test-library")
 include(":android-test:toast-rule")
 include(":android-test:snackbar-rule")
 include(":android-test:test-screenshot")
-include(":android-test:test-instrumentation-runner")
 include(":android-test:android-log")
 include(":android-test:rx3-idler")
 
 include(":android-lib:proxy-toast")
 include(":android-lib:snackbar-proxy")
 
+include(":test-runner:test-report-artifacts")
+include(":test-runner:test-annotations")
+include(":test-runner:test-report-api")
+include(":test-runner:test-report-dsl-api")
+include(":test-runner:test-report-dsl")
+include(":test-runner:test-report")
```

Comment:
How about grouping `test-report` modules?

```
test-report/
  api
  dsl-api
  dsl
  report
```

Reply:
not sure for now; let's draw a whole target picture before any changes (besides moving to test-runner xd)

Reply:
> let's draw a whole target picture before any changes

Agree. Is it applied to this PR also?

Reply:
not this PR